### DISCUSSION TITLE
Changes to .helm trigger rebuild and redeploy

### DISCUSSION THREAD

**Discussion by abrekhov** [werf/werf#6177] [General]
Hi! I’m using privileged gitlab  runner as k8s executor and werf v1.2

If i change helm templates, werf rebuilding docker image. In helm chart I'm using values.werf.image.appname reference. 

I want werf to pass the build stage because nothing changes.

**Comment by ilya-lesikov** [comment]
If you use Dockerfile: don't pass `.helm` into your images
If you use Stapel: make sure your stageDependencies do not match `.helm` or do not include `.helm` into your image in the first place.
